### ISSUE TITLE
Not a DOS Disk in device SD0

### ISSUE THREAD

**Issue by youenchene** [jbilander/SDBox-Mini#5]
Hi again,

Just need help on debugging.

I have one SDBox Mini working well.

I built 2 other one.

With the same amiga software, same SDCard and same firmware as the first working one on the 2 new ones I've got the this **Not a DOS Disk** error (french translating, i've got the message in french).

The orange LED is blinking when you insert the SD.

The error happen when you mount.

I've check all soldering, all resistor and all chips, all sounds OK.

Would be open to any clue on where to look.

Cheers again for your work.

**Comment by jbilander** [comment]
Hi, check that you have activity on the MOSI- and MISO-lines with a logic analyzer or oscope, compare with a working one, you might have a faulty IC or something. Also check the generated 16 MHz CLK looks good. And double-check soldering and pull-ups. Also you could try with the new driver that has Patrik's bugfix for better sd-card support, but since you are testing with a known good card it shouldn't be the culprit.
Here is a link to the latest spisd.device:
https://github.com/niklasekstrom/amiga-par-to-spi-adapter/pull/9

**Comment by youenchene** [comment]
New spisd.device test : same issue

I have to borrow a oscilloscope now.

**Comment by jbilander** [comment]
ok, yeah that was expected. If this is a Rev. 2B you are building I would suspect U4 (74AHCT1G126) is faulty or you forgot to remove the jumper after programming so MISO-signal is not working as it should, or perhaps the firmware wasn't programmed correctly. With Rev 2A you need to program the AVR before soldering down the 74LVC1G17 as it will otherwise interfere with the programming done via the ICSP-header, this problem was corrected in 2B.

**Comment by drphilxr** [comment]
I bought the SDBOX in 2022 from amigastore.eu and have the **exact same problem**
"Not a DOS disk" - tried FAT and FAt32 formats, reinstalled the SD0: files, the spisd.device (from the github, latest), and the 'Mount_SDCARD' icon does mount SD0:- just not a dos disk!
Drivers are in Devs: and fat95 (2013 version on aminet) in L:
USB power is on. Tried both 2Gb and 4Gb uSD cards.
im at my wits end. Tried on multiple A500's, from OS 2-3.2

**Comment by jbilander** [comment]
@drphilxr Them selling the product as a commercial item and then pawning off support to the community if something doesn't work is absolutely not good.

**Comment by drphilxr** [comment]
Yes agree completely- Amigastore.eu used to reply to emails about purchased products with support- not anymore. What can I say but its probably a uSD card issue, as per commentary here/elsewhere. Thankyou for hosting support questions!

**Comment by jbilander** [comment]
@drphilxr Did you try with the latest spisd.device from link on comment 2 above, thanks to Patrik the driver is more compatible with different MicroSD-cards. What you also could try is to put pull-up resistors on the Data-lines. We have seen on A3000 there is a need for pull-ups on the data-lines.
https://github.com/jbilander/sdbox/issues/18#issuecomment-1830798423

**Comment by drphilxr** [comment]
@jbilander thankyou for that; don't use sdbox on the A3000 now after investing $$ in a ZZ9000 card. But still need SDBox on an A500! So yeah using the same SD card (formatted Fat32- which Amigastore.eu states will work on their product page!)- and spisd.device v2- still "not a dos disk". Will of course try to format this card, and new ones on the way, with FAT (fat95) and see what happens. It's a journey...

**Comment by jbilander** [comment]
@drphilxr No problem, I think it's best if you do the formatting on a PC like this:
https://github.com/jbilander/SDBox-Mini/issues/2#issuecomment-1221970894
The 4GB restriction is no longer in play since Patrik's bugfix, but you might wanna keep it small(er) anyway because a larger partition takes more memory in AmigaOS.

**Comment by drphilxr** [comment]
So- did run CMD as admin, and disk part to clean (like cleaning the CF cards of old!), and did FAT formatting on a new 2Gb card. 
SPISD.device v2 installed in devs:, with SD0. WB 2.1. Click on "mount_sdcard"...
And.... 'Not a DOS disk" :(
The SDBox was working before on another A500. Time to ask Amigastore.eu if i can get a refund? Wish me luck - lol.
thankyou for your time also
